rest/client.py

```diff
@@ -1,4 +1,5 @@
 import requests
+import typing
```

Comment:
You still need to format imports (flake8 directive).

If you use pycharm, ctrl + alt +o should format you imports properly.
(Not sure If I told correct shortcut for windows, but its control + option + O on mac).